grails-app/domain/au/org/ala/specieslist/MatchedSpecies.groovy

```diff
@@ -31,8 +31,6 @@ class MatchedSpecies {
     String genus
     Date lastUpdated
 
-    static belongsTo = [speciesListItem:SpeciesListItem]
```

Comment:
Should we keep this relationship?

Reply:
We need to remove it.  'BelongsTo' will reverse the order of connection -> creates a species_list_item_id in matchedSpecies table.  It slows down the reading speed of species_list_item

Reply:
Ah sorry, just saw the change for SpeciesListItem